dialect/sql/schema/migrate.go

```diff
@@ -141,7 +150,15 @@ func (m *Migrate) create(ctx context.Context, tx dialect.Tx, tables ...*Table) e
 				return err
 			}
 		default: // !exist
+			// special case: sqllite driver create foreigh keys during table creating period,
+			// we need to set foreigh keys empty before it, and get FKS back after Query().
+			bks := t.ForeignKeys
+			if _, ok := m.sqlDialect.(*SQLite); ok && !m.withForeighKeys {
+				t.ForeignKeys = nil
+			}
```

Comment:
This logic is not belong here. Move this configuration to the SQLite driver creation.

Reply:
has been moved to sqlite.